baselines/fedmix/fedmix/conf/config.yaml

```diff
@@ -0,0 +1,23 @@
+defaults:
+  - dataset: cifar10
+  - strategy: fedmix
+  - client: ${strategy}
+  - model: vgg
+
+seed: 42
+server_device: cuda
+num_clients: 60
+clients_per_round: 15
+num_rounds: 500
+
+client_resources:
+  num_cpus: 4
+  num_gpus: 0.25
+
+hydra:
+  run:
+    dir: results/${dataset.name}_${hydra.job.num}
```

Comment:
The code won't start because `hydra.job.num` isn't defined when you set `hydra.run.dir`? (same comment applies to the `sweep` below. Where you able to run the code?

Reply:
One option would be to use the date/time as identifier:

```
hydra:
  run:
    dir: ./results/${dataset.name}/${now:%Y-%m-%d}/${now:%H-%M-%S} 
```

Reply:
I did this to generate the tables and figures neatly, with dates as the identifier, the user had to pass in path arguments for table/figures. `hydra.job.num` is automatically defined when hydra is in the `multirun` state, so it works in the `sweep` (`multirun`). Shall I change `dir` just for the normal `run` or do I use the date/time identifier?